### PR TITLE
Fix format secrets fix

### PR DESCRIPTION
**Pull request by mikepw**
re-introduces fix for Ansible grabbing config for TF out of keyvault.

### CODE REVIEW

File: script/tf_plan.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+if [ -z "$DEPLOY_TAG" ] || [ -z "$TF_DIR" ] || [ -z "$VAULT_SECRET" ] || [ -z "$VAULT_URL" ] || [ -z "$VAULT_CLIENT_ID" ] ||  [ -z "$VAULT_TENANT" ] ||  [ -z "$SUBSCRIPTION_ID" ]
+then
+echo "Set DEPLOY_TAG, TF_DIR, VAULT_SECRET, VAULT_CLIENT_ID, VAULT_TENANT, SUBSCRIPTION_ID"
+exit 1;
+fi
+
+cd ../ansible
+poetry run ansible-playbook ../ansible/site.yml --extra-vars "provision_pwdev=true deploy_tag=$DEPLOY_TAG tf_dir='$TF_DIR' vault_url='$VAULT_URL' vault_secret='$VAULT_SECRET' vault_client_id='$VAULT_CLIENT_ID' vault_tenant='$VAULT_TENANT' vault_subscription_id='$SUBSCRIPTION_ID'"
+
+#terraform show $TF_DIR/plan.tfplan
```
Comment (dandds) [inline script/tf_plan.sh:23]: Can we uncomment this here? Also, it complains about the provider not being initialized. If I update it to:

```
cd $TF_DIR
terraform show ./plan.tfplan
```
it works.

Comment (mikepw) [reply script/tf_plan.sh]: done

Comment (mikepw) [reply script/tf_plan.sh]: done and tested

File: script/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* Set env var `TF_DIR` which points to where your terraform lives
+* Set the `DEPLOY_TAG` to the semver version that correlates w/ the terraform you're executing 
+
+
+### PR Review Use Case
+
+* Follow the config steps above, the `deploy_tag` format is TBD but for now its based on semver.
+* Run the script
+* The result should not fail and you should be able to see the `plan.tfplan` output as a diff to what terraform would CRUD if applied.
+* Compare the `plan.tfplan` output to the intended changes from the PR comments
```
Comment (graham-dds) [inline script/README.md:29]: If this is PR specific, should it live in the PR description and not in this Readme?

File: script/tf_plan.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export PATH="$HOME/.poetry/bin:$PATH"
+export PYENV_ROOT="$HOME/.pyenv"
+export PATH="$PYENV_ROOT/bin:$PATH"
+if command -v pyenv 1>/dev/null 2>&1; then
+  eval "$(pyenv init -)"
+fi
+eval "$(pyenv virtualenv-init -)"
```
Comment (graham-dds) [inline script/tf_plan.sh:9]: Does this script require pyenv to run? If so, should that be outlined in the README? Also, what is the goal here, ultimately? Just to make sure a virtual environment exists? Could poetry be used for that instead? Just one less thing to manage.

File: script/tf_plan.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
```
Comment (graham-dds) [inline script/tf_plan.sh:11]: What is the thing that we need in `node_modules` for this script?

File: script/README.md (hunk 2 above)
Comment (mikepw) [reply script/README.md:29]: should be in read me.. as these are the steps to review future PRs (for now) that have to do w/ infra.

File: script/tf_plan.sh (hunk 4 above)
Comment (mikepw) [reply script/tf_plan.sh:11]: good call. its not related to this pr

File: script/tf_plan.sh (hunk 3 above)
Comment (mikepw) [reply script/tf_plan.sh:9]: ah copy/pasted my bash_profile.. will remove for you guys who may have different setups.

**Pull request merged**